### ISSUE TITLE
Сreate CRUD matrix

### ISSUE THREAD

**Issue by elizaveta-titova** [elizaveta-titova/ADM-Project-Restaurant-food-delivery#5]

**Comment by alenkaseledka** [comment]
Based on the description of the business process and the precedents diagram, these roles were identified:

Client
Administrator
Kitchen
Courier

**Comment by alenkaseledka** [comment]
Data were highlighted

Order
Dish
Admin data
Client data

**Comment by alenkaseledka** [comment]
Activities/processes were selected from activity diagrams:

Create new order
Add dishes to order
Update order list
Receive order
Calculate price
Receive personal client data
Create new order
Update client data
Decline order
Cook order'
Update order status
Deliver order
Get payment
Take order

**Comment by alenkaseledka** [comment]
grouped activities


Order
--
Create new order
Add dishes to order
Update order list
Receive order
Calculate order price
Cook order'
Update order status
Decline order
Deliver order
Take order
Get payment
Client Data
Add client data
Receive personal client data
Admin Data
Sign in
Create dish
Update dish
Delete dish

**Comment by alenkaseledka** [comment]
CRUD activities to roles 
<google-sheets-html-origin><style type="text/css"></style>

  | Client | Administrator | Kitchen | Courier
-- | -- | -- | -- | --
Create new order | CRU |   |   |  
Add dishes to order | CRU |   |   |  
Update order list | CRUD | RUD | R |  
Receive order | R | RU | R | R
Calculate order price | R | CRUD |   | R
Cook order' |   |   | CRUD |  
Update order status | R | RU | RU | R
Decline order | RD | RD | R |  
Deliver order | R | R |   | CRUD
Take order | RU | R |   | R
Get payment | RU |   |   | CRUD

**Comment by alenkaseledka** [comment]
CRUD matrix in google tab https://docs.google.com/spreadsheets/d/1MCKAtOYBtlH-Oz7pqLEWTEKDz5aVY92G0yNKJa3bDD0/edit?hl=ru#gid=298939070

**Comment by fuljencia** [comment]
@alenkaseledka , let's get rid of the **activities** about dish and admin, as it's not really about our main process of delivery. We may get more confused with them

**Comment by fuljencia** [comment]
@alenkaseledka , I also suggest to discuss a bit the naming in the table "activities to roles" on our future zoom-call

**Comment by fuljencia** [comment]
@alenkaseledka , by the way, "data to roles" is great!

**Comment by elizaveta-titova** [comment]
@fuljencia @alenkaseledka  I actually agree with Alisa that we can delete some activities related to Admin and dish, so it'll be easier to interpret the results.

**Comment by elizaveta-titova** [comment]
@fuljencia Yes, we can discuss the names of activities and maybe create something more while brainstorming together.

**Comment by alenkaseledka** [comment]
sure, let's discuss it together!

**Comment by fuljencia** [comment]
I suggest to add following resources: 
- computers
- order sheet (kitchen uses it for the order cooking)

**Comment by alenkaseledka** [comment]
CRUD for Resources to Roles

<google-sheets-html-origin><style type="text/css"></style>

  | Client | Administrator | Kitchen | Courier
-- | -- | -- | -- | --
Computers | CRUD | CRUD |   |  
Order sheet |   | CRUD | R | R
Courier's phone |   |   |   | CRUD

**Comment by alenkaseledka** [comment]
On a call we decided that these CRUD matrix are the final versions
